core/shared/src/main/scala/zio/metrics/CurrentMetrics.scala

```diff
@@ -0,0 +1,94 @@
+package zio.metrics
+
+import zio.internal.metrics.metricRegistry
+import zio.metrics.CurrentMetrics.MetricsRenderer
+import zio.{IO, Trace, UIO, ZIO}
+
+import java.io.IOException
+
+final case class CurrentMetrics private[metrics] (metrics: Set[MetricPair.Untyped]) extends Serializable {
+  def prettyPrint(implicit trace: Trace): UIO[String] =
+    MetricsRenderer.prettyPrint(metrics)
+}
+
+object CurrentMetrics {
+
+  /**
+   * Gets current metrics snapshot.
+   */
+  def snapshot(): UIO[CurrentMetrics] =
+    ZIO.succeedUnsafe { implicit u =>
+      CurrentMetrics(metricRegistry.snapshot())
+    }
+
+  /**
+   * Dumps all current metrics to the specified callback.
+   */
+  def dumpWith[R, E, T](f: CurrentMetrics => ZIO[R, E, T])(implicit trace: Trace): ZIO[R, E, T] =
```

Comment:
This should not exist because a user can always `flatMap` a `Metrics`, having acquired it with `ZIO.metrics`.